tools/ci/run_tc.py

```diff
@@ -169,15 +172,16 @@ def install_webkitgtk_from_apt_repository(channel):
     run(["sudo", "apt-get", "-qqy", "-t", "bionic-wpt-webkit-updates", "install", "webkit2gtk-driver"])
 
 
-# Download an URL in chunks and saves it to a file descriptor (truncating it)
-# It doesn't close the descriptor, but flushes it on success.
-# It retries the download in case of ECONNRESET up to max_retries.
 def download_url_to_descriptor(fd, url, max_retries=3):
+    """Download an URL in chunks and saves it to a file descriptor (truncating it)
+       It doesn't close the descriptor, but flushes it on success.
```

Comment:
Nit: add a blank line before this line, and remove 1 level of indentation.

Reply:
An initial blank line doesn't match the style of other code afaict?